### ISSUE TITLE
Test-extended.functional-JDK11-linux_x86-64 TestJava9AttachAPI_0 FAILED: testAllowAttachSelfBareEnable

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#2935]
https://ci.eclipse.org/openj9/job/Test-extended.functional-JDK11-linux_x86-64/20/tapResults/

```
===============================================
Running test TestJava9AttachAPI_0 ...
===============================================
TestJava9AttachAPI_0 Start Time: Wed Sep 19 08:59:21 2018 Epoch Time (ms): 1537347561868
test with NoOptions
[IncludeExcludeTestAnnotationTransformer] [INFO] exclude file is /home/jenkins/workspace/Test-extended.functional-JDK11-linux_x86-64/openjdk-tests/TestConfig/scripts/testKitGen/../../../../jvmtest/TestConfig/resources/excludes/latest_exclude_11.txt
...
... TestNG 6.14.2 by Cédric Beust (cedric@beust.com)
...

PASSED: testAllowAttachSelfDefault
PASSED: testAllowAttachSelfDisable
PASSED: testAllowAttachSelfEnable
FAILED: testAllowAttachSelfBareEnable
java.lang.AssertionError: wrong exit status expected [13] but found [141]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:652)
	at org.openj9.test.attachAPI.TestJava9AttachAPI.testAllowAttachSelfBareEnable(TestJava9AttachAPI.java:74)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(java.base@11-internal/Native Method)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(java.base@11-internal/NativeMethodAccessorImpl.java:62)
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(java.base@11-internal/DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(java.base@11-internal/Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.TestNG.privateMain(TestNG.java:1354)
	at org.testng.TestNG.main(TestNG.java:1323)


===============================================
    TestJava9AttachAPI
    Tests run: 4, Failures: 1, Skips: 0
===============================================
```

**Comment by pshipton** [comment]
@pdbain-ibm fyi

**Comment by pdbain-ibm** [comment]
`wrong exit status expected [13] but found [141]`: Possible duplicate of issue #2871

**Comment by pshipton** [comment]
Seems to be intermittent as it didn't occur in the next build
https://ci.eclipse.org/openj9/job/Test-extended.functional-JDK11-linux_x86-64/21/tapResults

**Comment by pdbain-ibm** [comment]
```$ kill -l 13
PIPE
```
The VM received a SIGPIPE (signal 13) and exited with status code 128 + 13 = 141.

**Comment by pshipton** [comment]
Closing as fixed by #2965
